src/inspector/main_thread_interface.cc

```diff
@@ -288,11 +289,13 @@ Deletable* MainThreadInterface::GetObjectIfExists(int id) {
   return iterator->second.get();
 }
 
-std::unique_ptr<StringBuffer> Utf8ToStringView(const std::string& message) {
-  icu::UnicodeString utf16 = icu::UnicodeString::fromUTF8(
-      icu::StringPiece(message.data(), message.length()));
-  StringView view(reinterpret_cast<const uint16_t*>(utf16.getBuffer()),
-                  utf16.length());
+std::unique_ptr<StringBuffer> Utf8ToStringView(const std::string_view message) {
+  size_t expected_u16_length =
+      simdutf::utf16_length_from_utf8(message.data(), message.length());
+  char16_t buffer[expected_u16_length];
```

Comment:
The linter is giving warnings here. Only possible solution I can think of is creating a `std::vector` which will impact the performance. Any suggestions? @addaleax

Reply:
I know you already fixed this but for background, we don't use VLAs (variable-length arrays) because they're evil - it's very easy to run out of stack space and SIGBUS (or worse) on large inputs.